tests/New-DbaDbTable.Tests.ps1

```diff
@@ -56,4 +57,23 @@ Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
             $table.Columns.DefaultConstraint.Name | Should -Contain "DF_MyTest"
         }
     }
+    Context "Should create the table with an identity column" {
+        It "Creates the table" {
+            $map = @{
+                Name              = 'testId'
+                Type              = 'int'
+                Identity          = $true
+                IdentitySeed      = 10
+                IdentityIncrement = 2
+            }
+            (New-DbaDbTable -SqlInstance $script:instance1 -Database $dbname -Name $tablename3 -ColumnMap $map).Name | Should -Contain $tablename3
+        }
+        It "Has an identity column" {
+            $table = Get-DbaDbTable -SqlInstance $script:instance1 -Database $dbname -Table $tablename3
+            $table.Name | Should -Be $tablename3
+            $table.Columns.Identity | Should -Be $true
+            $table.Columns.IdentitySeed | Should -Be $map.IdentitySeed
```

Comment:
The test value is not being picked up so it failed. Adjusting the above `map` object to be in a `BeforeEach` I think will make it more consistent.